src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Automation/VsBuildManager.cs

```diff
@@ -30,13 +32,16 @@ internal VSBuildManager(IProjectThreadingService threadingService, IUnconfigured
         {
             AddEventSource(this);
             _threadingService = threadingService;
-
             Project = new OrderPrecedenceImportCollection<VSLangProj.VSProject>(projectCapabilityCheckProvider: unconfiguredProjectServices.Project);
+            DesignTimeInputsBuildManagerBridge = new OrderPrecedenceImportCollection<IDesignTimeInputsBuildManagerBridge>(projectCapabilityCheckProvider: unconfiguredProjectServices.Project);
         }
 
         [ImportMany(ExportContractNames.VsTypes.VSProject)]
         internal OrderPrecedenceImportCollection<VSLangProj.VSProject> Project { get; }
 
+        [ImportMany]
+        internal OrderPrecedenceImportCollection<IDesignTimeInputsBuildManagerBridge> DesignTimeInputsBuildManagerBridge { get; }
```

Comment:
The contract this inteface is marked with is marked as ExactlyOne but you import it as a collection?

Reply:
This is probably just my lack of knowledge of MEF. I can't import the class directly because it would be a circular reference, and the capabilities of IDesignTimeInputsBuildManagerBridge are more restricted than VsBuildManager, so I was under the impression I need to use OrderPrecedenceImportCollection to do the capabilities check. Is there an alternative approach?